x-pack/plugins/security_solution/public/app/home/setup.tsx

```diff
@@ -39,13 +39,9 @@ export const Setup: React.FunctionComponent<{
       });
     };
 
-    if (!ingestManager.success) {
-      if (ingestManager.error) {
-        displayToastWithModal(ingestManager.error.message);
-      } else {
-        displayToast();
-      }
-    }
+    ingestManager.success
+      .then(displayToast)
```

Comment:
This does not look right given the prior implementation. Success should do nothing - so no `.then()` block. Only the `.catch()` should process the error, and based on the new implementation where I think we always get an `Error` object, I think that perhaps the `displayToast()` callback is no longer needed.

Reply:
Yes, I see the flaw now. Thanks for the :eyes: